lib/player.rb

```diff
@@ -0,0 +1,60 @@
+require 'awesome_print'
+require_relative '../lib/scoring.rb'
+require 'pry'
+module Scrabble
+  class Player
+    attr_reader :name, :plays, :total
+    attr_accessor :tiles
+    def initialize(name)
+      @name = name
+      @plays = Array.new
+      @total = 0
+      @tilebag = Scrabble::Tilebag.new
```

Comment:
Each player should **NOT** receive their own TileBag.  Instead use the parameter provided to `draw_tiles`.